gluoncv/data/transforms/mask.py

```diff
@@ -125,22 +131,31 @@ def fill(mask, bbox, size):
     # expand boxes
     x1, y1, x2, y2 = bbox
     x, y, hw, hh = (x1 + x2) / 2, (y1 + y2) / 2, (x2 - x1) / 2, (y2 - y1) / 2
-    hw = hw * ((M + 2) * 1.0 / M)
-    hh = hh * ((M + 2) * 1.0 / M)
+    hw = hw * (float(M + 2) / M)
+    hh = hh * (float(M + 2) / M)
     x1, y1, x2, y2 = x - hw, y - hh, x + hw, y + hh
-    # quantize
-    x1, y1, x2, y2 = map(int, (x1, y1, x2, y2))
-    w, h = (x2 - x1 + 1), (y2 - y1 + 1)
+    if fast_fill:
+        # quantize
+        x1, y1, x2, y2 = map(int, (x1 + 0.5, y1 + 0.5, x2 + 0.5, y2 + 0.5))
+        w, h = (x2 - x1 + 1), (y2 - y1 + 1)
+        mask = mx.nd.array(mask)
```

Comment:
change to use PIL resize directly?

Reply:
is there any issue with mx imresize?

Reply:
nothing bad with imresize, but numpy.ndarray->NDArray->numpy.ndarray requires two memcpy